src/objects/rhythm/rhythm.stories.mdx

```diff
@@ -8,132 +9,125 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
-    name="Default"
-    parameters={{ docs: { source: { code: introDemoSource } } }}
-  >
-    {introDemo}
-  </Story>
+  <Story name="Example">{(args) => demo(args)}</Story>
 </Canvas>
 
-## Modifiers
+<ArgsTable story="Example" />
 
-For shorter summary content, use the `o-rhythm--condensed` modifier to tighten things up a bit:
+## Amount
 
-<Canvas>
-  <Story
-    name="Condensed"
-    parameters={{ docs: { source: { code: summaryDemoSource } } }}
-  >
-    {summaryDemo}
-  </Story>
-</Canvas>
+The default amount of space works well for medium-to-long text content, but you can customize the amount of space for different situations by specifying the `amount` property or a class name of `o-rhythm--{amount}` with one of the following keywords:
 
-`o-rhythm--compact` is best for shorter content chunks that just need a _pinch_ more space:
+- `compact`: For short chunks of summary content, such as a heading, sentence and button.
+- `condensed`: For short lists of content that need a bit of space between.
+- `default`: Reset to the default amount of space, useful for nested rhythm objects.
+- `generous`: Apply enough space for each element to feel distinct from its siblings.
+- `lavish`: Apply the same amount of vertical space as padded [containers](/?path=/docs/objects-container--basic) have for consistent spacing across separate sections.
 
 <Canvas>
-  <Story
-    name="Compact"
-    parameters={{ docs: { source: { code: listDemoSource } } }}
-  >
-    {listDemo}
+  <Story name="Amount" args={{ amount: 'condensed' }}>
+    {(args) => demo(args)}
   </Story>
 </Canvas>
 
-While `o-rhythm--generous` is intended for separating distinct content chunks with more whitespace:
+## Heading Amount
 
-<Canvas>
-  <Story
-    name="Generous"
-    parameters={{ docs: { source: { code: paragraphsDemoSource } } }}
-  >
-    {paragraphsDemo}
-  </Story>
-</Canvas>
+It can be helpful to differentiate headings with more preceding space than normal flow content. In those cases, you can specify a `heading_amount` property or a class name of `o-rhythm--{heading_amount}-headings` with one of the following keywords:
 
-And finally, `o-rhythm--lavish` can separate entire sections from each other. It uses the same fluid padding values as [the container object](/?path=/docs/objects-container--basic):
+- `generous`
+- `lavish`
 
 <Canvas>
   <Story
-    name="Lavish"
-    parameters={{
-      layout: 'fullscreen',
-      docs: { source: { code: sectionsDemoSource } },
-    }}
+    name="Heading Amount"
+    args={{ amount: 'condensed', heading_amount: 'generous' }}
   >
-    {sectionsDemo}
+    {(args) => demo(args)}
   </Story>
 </Canvas>
 
-To revert to the default spacing in nested rhythm objects, use the `o-rhythm--default` modifier.
+## Inheriting Rhythm
 
-## Heading Modifiers
+The vertical spacing applied by the Rhythm object only applies to direct children. In some cases, you may want elements below to inherit rhythm. This can be accomplished in two main ways.
```

Comment:
```suggestion
The vertical spacing applied by the Rhythm object only applies to direct children. In some cases, you may want elements below to inherit rhythm. This can be accomplished in two main ways, by nesting Rhythm objects or via Sass.
```
And then below, you can change the heading "Via Sass" to only "Sass".

Up to you on this one. I wanted the transition to connect a bit more so I made this suggestion. 🙂